### DISCUSSION TITLE
Disconnect Command

### DISCUSSION THREAD

**Discussion by dylanb-101** [TannerGabriel/discord-bot#128] [Ideas]
It would be nice to have a /disconnect command (or /dc as an alias) that would automatically disconnect the bot and clear the queue so I don't have to right-click and force disconnect the bot if I want it to stop mid song.

**Comment by TannerGabriel** [comment]
@dylanb-101 the `stop` command clears the queue and disconnects the bot, which should be precisely the functionality you want.

**Reply by dylanb-101** [reply]
I didn't know that I thought it only cleared the queue thanks!
